driver/razermousemat_driver.c

```diff
@@ -645,7 +660,8 @@ static int razer_mousemat_probe(struct hid_device *hdev, const struct hid_device
         goto exit;
     }
 
-    if(intf->cur_altsetting->desc.bInterfaceProtocol == USB_INTERFACE_PROTOCOL_MOUSE) {
+    if(intf->cur_altsetting->desc.bInterfaceProtocol == USB_INTERFACE_PROTOCOL_MOUSE
+       || intf->cur_altsetting->desc.bInterfaceProtocol == RAZER_FIREFLY_V2_INTERFACE_PROTOCOL) {
```

Comment:
Please adapt the code like in https://github.com/openrazer/openrazer/commit/5eac33fafb2ee4277b12c42e90c5982a2450427b#diff-6f312f76ec0e3d4e5da9170fe783258bR806

Reply:
I have added an if above this part so that when the idProduct == razer firefly v2 then I set the other USB protocol.
I haven't used a switch here because currently, only the v2 uses the other protocol and because v2 does have wave effect I need it to fall to the 'default' file creation part. That's why I kept the switch over the idProduct as it was

If you'd like me to adapt this to a switch let me know